rel-eng/openshift-node.sysconfig

```diff
@@ -1,2 +1,5 @@
 ROLE="node"
 OPTIONS="--loglevel=0"
+# If running node on a separate host please rsync the admin directory ie:
+# rsync root@openshift-master:/var/lib/openshift/openshift.local.certificates/admin /var/lib/openshift/openshift.local.certificates/
+KUBECONFIG="/var/lib/openshift/openshift.local.certificates/admin/.kubeconfig"
```

Comment:
I think that makes sense. Would it be crazy to suggest that `openshift start` be able to consume a config file?